activities.el

```diff
@@ -858,44 +859,44 @@ OLDEST-POSSIBLE is the oldest age in the `vc-annotate-color-map'."
     (when-let ((activity (map-elt activities-activities name)))
       (let (data)
 	(dolist (type '(last default))
-	  (let* ((state (cl-struct-slot-value 'activities-activity type activity))
-		 (time (map-elt (activities-activity-state-etc state) 'time))
-		 (window-state (activities-activity-state-window-state state))
-		 (buffers (window-state-buffers window-state))
-		 (files (activities--map-window-state-leafs
-			 window-state
-			 (lambda (leaf)
-			   (bookmark-get-filename
-			    (activities-buffer-bookmark
-			     (map-nested-elt (cdr leaf)
-			 		     '(parameters activities-buffer))))))))
-	    (setf (alist-get type data)
-		  (list (length buffers)
-			(length (delq nil files))
-			(float-time (time-since time))))))
-	(pcase-let* ((`(,last-bufs ,last-files ,last-time) (map-elt data 'last))
-		     (`(,default-bufs ,default-files ,default-time) (map-elt data 'default))
-		     (age (min last-time default-time))
-		     (ann (format "%s:%s|%s %s:%s|%s "
-				  (propertize "buffers" 'face 'bold)
-				  (propertize (format "%2d" last-bufs) 'face 'success)
-				  (propertize (format "%-2d" default-bufs) 'face 'warning)
-				  (propertize "files" 'face 'bold)
-				  (propertize (format "%2d" last-files) 'face 'success)
-				  (propertize (format "%-2d" default-files) 'face 'warning)))
+	  (when-let ((state (cl-struct-slot-value 'activities-activity type activity)))
+	    (let* ((time (map-elt (activities-activity-state-etc state) 'time))
+		   (window-state (activities-activity-state-window-state state))
+		   (buffers (window-state-buffers window-state))
+		   (files (activities--map-window-state-leafs
+			   window-state
+			   (lambda (leaf)
+			     (bookmark-get-filename
+			      (activities-buffer-bookmark
+			       (map-nested-elt (cdr leaf)
+			 		       '(parameters activities-buffer))))))))
+	      (setf (alist-get type data)
+		    (list (length (delq nil files))
+			  (and time (float-time (time-since time))) buffers)))))
+	(pcase-let* ((`(,last-file-cnt ,last-age ,last-bufs) (map-elt data 'last))
```

Comment:
Forgive the nitpicking, but I'd still prefer not to abbreviate these variable names.  I'd suggest, e.g. `num-last-files`, `last-buffers`, etc.

Reply:
Not at all!  You'll look at this more than me, so whatever names area easiest on your eyes is preferred.